### PR TITLE
ARTEMIS-2582: EmbeddedActiveMQ.stop() should check for null

### PR DESCRIPTION
**Pull request by nbrendah**
PR is created to as response for https://issues.apache.org/jira/browse/ARTEMIS-2582

cc @clebertsuconic

### CODE REVIEW

File: artemis-server/src/main/java/org/apache/activemq/artemis/core/server/embedded/EmbeddedActiveMQ.java [hunk 1]
```diff
@@ -139,7 +139,9 @@ protected void initStart() throws Exception {
    }
 
    public EmbeddedActiveMQ stop() throws Exception {
-      activeMQServer.stop();
+      if (activeMQServer != null) {
+         activeMQServer.stop();
+      }
       return this;
```
[the remark this answers is not in the document]
Comment (nbrendah) [reply artemis-server/src/main/java/org/apache/activemq/artemis/core/server/embedded/EmbeddedActiveMQ.java:145]: hello @clebertsuconic 
do you have any suggestions or comments on this?

**Comment by nbrendah** [comment]
hello @clebertsuconic
do you have any suggestions or comments on this?

**Comment by gtully** [comment]
it may as well have a little test, to ensure there is no regression in the future

**Comment by jbertram** [comment]
I added a test during merge. Thanks, @nbrendah!

**Pull request closed without merging**